helpers.py

```diff
@@ -120,9 +120,13 @@ def fio_dnsbase():
     '''Find the dns base hosting jobserv. For example when running under
        api.foundries.io, this would return foundries.io
     '''
-    parts = urlparse(os.environ["H_RUN_URL"])
-    _, base = parts.netloc.split(".", 1)
-    return base
+    run_url = os.environ.get("H_RUN_URL")
```

Comment:
Could be `os.environ.get("H_RUN_URL", "https://api.foundries.io"`) too. Or, `os.environ.get("H_RUN_URL",os.environ.get("GIT_URL")`)

Reply:
I see it in the bitbake runs.

```
GIT_URL=https://source.foundries.io/factories/msul-dev01/lmp-manifest.git
```